config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -195,58 +195,37 @@ periodics:
-- interval: 1h
-  name: ci-kubernetes-e2e-gci-gce-scalability-containerd
-  tags:
-    - "perfDashPrefix: gce-100Nodes-master-containerd"
-    - "perfDashJobType: performance"
-    - "perfDashBuildsCount: 500"
+  # Experimental job providing visibility into state of containerd.
+- interval: 6h
+  name: ci-kubernetes-e2e-gci-gce-scalability-containerd-conformance
   labels:
     preset-service-account: "true"
     preset-k8s-ssh: "true"
     preset-e2e-scalability-common: "true"
-    preset-e2e-scalability-containerd: "true"
   annotations:
     testgrid-dashboards: sig-scalability-experiments
-    testgrid-tab-name: gce-cos-master-scalability-100-containerd
+    testgrid-tab-name: gce-cos-master-scalability-100-containerd-conformance
   spec:
     containers:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20200602-05eeaff-master
         args:
-          - --timeout=140
           - --repo=k8s.io/kubernetes=master
-          - --repo=k8s.io/perf-tests=master
           - --root=/go/src
           - --scenario=kubernetes_e2e
+          - --timeout=140
           - --
           - --check-leaked-resources
-          - --cluster=e2e-big-containerd
-          # TODO(oxddr): remove once debugging is finished
-          - --env=KUBEPROXY_TEST_LOG_LEVEL=--v=4
+          - --cluster=e2e-containerd-conformance
           - --extract=ci/latest
           - --gcp-node-image=gci
           - --gcp-nodes=100
           - --gcp-project-type=scalability-project
           - --gcp-zone=us-east1-b
           - --provider=gce
-          - --test=false
-          - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
-          - --test-cmd-args=cluster-loader2
-          - --test-cmd-args=--experimental-gcp-snapshot-prometheus-disk=true
-          - --test-cmd-args=--experimental-prometheus-disk-snapshot-name=${JOB_NAME}-${BUILD_ID}
-          - --test-cmd-args=--nodes=100
-          - --test-cmd-args=--prometheus-scrape-node-exporter
-          - --test-cmd-args=--provider=gce
-          - --test-cmd-args=--report-dir=/workspace/_artifacts
-          - --test-cmd-args=--testconfig=testing/density/config.yaml
-          - --test-cmd-args=--testconfig=testing/load/config.yaml
-          - --test-cmd-args=--testconfig=testing/access-tokens/config.yaml
-          - --test-cmd-args=--testoverrides=./testing/experiments/enable_prometheus_api_responsiveness.yaml
-          - --test-cmd-args=--testoverrides=./testing/experiments/enable_restart_count_check.yaml
-          - --test-cmd-args=--testoverrides=./testing/experiments/use_simple_latency_query.yaml
-          - --test-cmd-args=--testoverrides=./testing/overrides/load_throughput.yaml
-          - --test-cmd-name=ClusterLoaderV2
+          - --test=true
+          - --ginkgo-parallel=40
+          - --test_args=--ginkgo.focus=\[Conformance\] --ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[DisabledForLargeClusters\]
```

Comment:
What is the reason for making that conformance and not what we run in all other correctness tests?

Reply:
I would like to slowly roll out containerd to full correctness suite (one that we run for docker), when it will have enough soaking time.

Starting as an experimental job with a limited set of tests seams like a reasonable point to start.

Reply:
Why can't we just make it consistent with all other correctness jobs. Given it's experimental, it's not big deal if it will start failing. And we will be able to uncover and delegate more problems at once when we find them.

Reply:
Removed `focus`, making it match other correctness jobs.